content/learn/fundamental-wellbeing.md

```diff
@@ -1,73 +1,96 @@
 ---
-title: "Fundamental Wellbeing"
-created: 2023-04-21
-authors: 
+title: An Introduction to Dr. Jeffery Martin's Work on Fundamental Wellbeing
+created: 2023-11-01
+authors:
   - rufuspollock
   - jefferymartin
```

Comment:
I wouldn't list jeffery as an author here. We wrote the page, even if we quote him etc.

Reply:
I removed Jeffery